readme.md

```diff
@@ -0,0 +1,47 @@
+[![Run PHPUnit and PHPCS](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml/badge.svg)](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml)
+
+# Maintenance Mode
+
+Shut down your site for a little while and do some maintenance on it!
+
+Usage:
+
+ - Add a template to your theme's root folder called `template-maintenance-mode.php`.
```

Comment:
Is this required or optional?

Reply:
I think this should be spun off into a separate issue, then update both readmes with any instructional improvements